### ISSUE TITLE
[Discussion] Kavita+ - Removal of GoogleBooks

### ISSUE THREAD

**Issue by majora2007** [Kareadita/Kavita#2493]
Google Books support has always been lacking within Kavita+. In the effort to drive more metadata, I tried to use GoogleBooks for ratings, but after it being out for a few months, I do not believe the results are consistent enough and add much value. 

I am proposing to remove this feature in the short term. I may revive when I implement the [individual rating](https://discord.com/channels/821879810934439936/1179931711585386566) feature request, as books need individual ratings. 

I am open to discussion from the community. From the initial polling in the discord of Kavita+ users, it doesn't seem to be widely used.

**Comment by therobbiedavis** [comment]
For Kavita's purpose, Google books seems woefully lacking. It is a good source for details about the book, but lacks ratings or any other user based data.

I came across a new service that has been called "the letterboxd for books", https://hardcover.app which does have a good amount of rating, at least for more popular books.

I looked and they do have an api available once you sign up (https://hardcover.app/account/api), with plans to expand and improve it. They also have an active discord which would be good for developer collaboration.

![image](https://github.com/Kareadita/Kavita/assets/7907265/72287842-5c06-48c5-acac-83efb09aadc9)
